### DISCUSSION TITLE
Lazer updates reset Windows Volume Mixer preferences

### DISCUSSION THREAD

**Discussion by maxmine2** [ppy/osu#28623] [Q&A] [answered]
By the recommendation of a fellow forum member, I duplicate this request from here [link to forum post in Help](https://osu.ppy.sh/community/forums/topics/1940385)

Every time after an update of osu!(lazer) is installed, I'm met with a problem that Volume Mixer preferences for osu!(lazer) are reset (i. e. if the default loudness is set for 60 and osu! is set for 15, after an update it will go back to default and become 70, giving a good scare with a rapid loudness increase)

This is not a critical vulnerability nor a serious issue. This is easily fixable issue for user, as all it takes to fix is to just turn the volume back to its initial loudness. However it might deliver an unpleasant experience for unprepared for the rapid volume increase user, especially if the loudness growth might be a quite a serious threat (apart from making someone ears hurt from the "Welcome to osu!" phrase).

This might also be connected to how Windows 10 treats osu! and its updates, so there might be not much one can do rather than prepare for sudden loudness increase

### Settings how they looked before the update (recreation):

![image](https://github.com/ppy/osu/assets/44315846/9ea93228-5ab7-444c-bbab-89b2e05cfc80)

### Settings after the update is applied and osu! restarts

![image](https://github.com/ppy/osu/assets/44315846/ebf84632-621a-4503-bec2-78947ff317e9)

As you can see, the loudness preference goes to the maximum allowed (default state for new applications). This might be due to how osu! works with updates, since this problem does not occur after other applications update.

osu! version: osu!(lazer) 2024.625.2
OS version: Windows 10 Pro (10.0.19045 build 19045)

Logs file are related to today (26.06.2024), since the update was applied today.
[logs.zip](https://github.com/user-attachments/files/15991259/logs.zip)

I have not found any other discussions related to the same topic, as this might be not a very important problem, however it might still be one. I would be glad to answer any questions related to this issue.

**Comment by peppy** [comment, marked as answer]
This will be fixed when we get to updating the updater, which means the executable doesn't move around each update.
